recipe/0125-rendering-3d/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Including a 3D Rendering
+id: 93
+layout: recipe
+tags: [tbc]
+summary: "You want to include a 3D model representation"
+---
+
+
+## Use Case
+
+You want to view an object from any orbital point of view or zoom level.
+You want to view an object in augmented reality/XR in order to gain a better understanding of its dimensions and physical properties.

```

Comment:
Could you give more details on the use case. This is a good example to follow: https://iiif.io/api/cookbook/recipe/0007-string-formats/. Its best to assume the reader is a novice with IIIF.